cql3/functions/user_function.cc

```diff
@@ -7,6 +7,8 @@
  */
 
 #include "user_function.hh"
+#include "lang/udf_cache.hh"
+#include "lang/wasmtime.hh"
```

Comment:
Let's work to have an interface to engines and so only include the abstract engine, not every runtime here. Not high priority but keep in mind.